File: app/src/main/java/com/blogspot/thengnet/mobilestudent/Audio.java
package com.blogspot.thengnet.mobilestudent;

public class Audio {

    private String mAudioTitle;
    private String mAudioLength;
    private String mAudioDisplayName;
    private String mAudioPath;

    public Audio () {
    }

    public Audio (String mAudioTitle, String mAudioDisplayName, String mAudioLength/*, String mAudioPath*/) {
        this.mAudioTitle = mAudioTitle;
        this.mAudioLength = mAudioLength;
        this.mAudioDisplayName = mAudioDisplayName;
        this.mAudioPath = mAudioPath;
    }

    public String getAudioTitle () {
        return mAudioTitle;
    }

    public String getAudioLength () {
        return mAudioLength;
    }

    public String getAudioDisplayName () {
        return mAudioDisplayName;
    }

    public String getAudioPath () {
        return mAudioPath;
    }
}

File: app/src/main/java/com/blogspot/thengnet/mobilestudent/Audio.kt
package com.blogspot.thengnet.mobilestudent;

public class Audio {

    private String mAudioTitle;
    private String mAudioPath;
    private String mAudioLength;

    public Audio () {
    }

    public Audio (String theAudioTitle, String theAudioPath, String theAudioLength) {
        this.mAudioTitle = theAudioTitle;
        this.mAudioPath = theAudioPath;
        this.mAudioLength = theAudioLength;
    }

    public String getAudioTitle () {
        return mAudioTitle;
    }

    public String getAudioLength () {
        return mAudioLength;
    }

    public String getAudioPath () {
        return mAudioPath;
    }
}